src/OrchardCore.Modules/OrchardCore.Spatial/Views/GeoPointField.Edit.cshtml

```diff
@@ -0,0 +1,25 @@
+@using OrchardCore.Spatial.ViewModels
+@using OrchardCore.Spatial.Settings
+@model EditGeoPointFieldViewModel
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<GeoPointFieldSettings>();
+}
+
+<div class="form-row">
+    <div class="form-group col-md-4">
+        <label asp-for="Latitude">Latitude</label>
+        <input asp-for="Latitude" type="number" min="-90.000000" max="90.000000" step=".000001" placeholder="00.000000" class="form-control content-preview-text" required="@settings.Required" />
+        @if (!string.IsNullOrEmpty(settings.Hint))
+        {
+            <span class="hint">@settings.Hint</span>
+        }
+    </div>
+    <div class="form-group col-md-4">
+        <label asp-for="Longitude">Longitude</label>
+        <input asp-for="Longitude" type="number" min="-180.000000" max="180.000000" step=".000001" placeholder="00.000000" class="form-control content-preview-text" required="@settings.Required" />
```

Comment:
Would a `decimal` more adapted since the precision matters and no data should be lost?

Reply:
https://stackoverflow.com/questions/28068123/double-or-decimal-for-latitude-longitude-values-in-c-sharp

Reply:
Here the thing I would add to this is the fact that generally you won't need more than 6 digits precision.

decimal
places   degrees          distance
-------  -------          --------
| decimal places | degrees | distance |
|---|---|---|
|  0  | 1 |  111  km |
|  1  | 0.1 |  11.1 km  |
|  2  | 0.01 |  1.11 km  |
|  3  | 0.001 |  111  m  |
|  4  | 0.01 |  11.1 m  |
|  5  | 0.01 |  1.11 m  |
|  6  | 0.01 |  11.1 cm  |
|  7  | 0.01 |  1.11 cm  |
|  8  | 0.01 |  1.11 mm  |
|  9  | 0.01 |  111  μm  |
|  10  | 0.01 |  11.1 μm  |
|  11  | 0.01 |  1.11 μm  |
|  12  | 0.01 |  111  nm  |
|  13  | 0.01 |  11.1 nm  |

Reply:
What we are missing is a configurable setting on the Lucene settings to change this 11 maxlength precision. And on each of the fields we should also allow to set a maxlength and then adjust the editor UI in consequence.